scripts/terra_table_from_gcp_assemblies.sh

```diff
@@ -5,7 +5,7 @@ Will create a terra data table with samplenames and gcp pointers to assemblies b
 
 For the Terra table to properly import into the user-defined workspace, gcloud authentication is required. 
 
-Five positional arguments required:
+Seven positional arguments required:
```

Comment:
Five required and two optional, it seems.

Reply:
good catch, i'll push an update